src/chaat/handler/handler.clj

```diff
@@ -4,7 +4,14 @@
             [java-time.api :as jt]
             [chaat.handler.validation :as handler.validation]
             [chaat.errors :refer [do-or-error]]
-            [chaat.db.utils :as db.utils]))
+            [chaat.db.utils :as db.utils]
+            [cheshire.core :as json]))
+
+(defn send-response
+  [{:keys [result error]}]
+  (if-not error
+    (res/response (json/encode result))
+    (res/status (res/bad-request (json/encode (:msg error))) (:status-code error))))
```

Comment:
This line is a bit confusing. A [400 bad request](https://github.com/ring-clojure/ring/blob/1ee170e6b67e05484985e4245cc625c812bb22e1/ring-core/src/ring/util/response.clj#L47-L53) is being constructed and then being overridden it with some other status code. Semantically the intent would be more clear if we use `res/response` instead.
```suggestion
    (-> (res/response (json/encode (:msg error)))
        (res/status (:status-code error)))
```

Reply:
This is a great utility, btw

Reply:
Yes, this is definitely nicer and more logical. Done, thank you.